test/unit/org/apache/cassandra/db/filter/ColumnFilterTest.java

```diff
@@ -52,14 +52,10 @@
 
 import static org.junit.Assert.assertEquals;
 
-@RunWith(Parameterized.class)
 public class ColumnFilterTest
 {
     private static final ColumnFilter.Serializer serializer = new ColumnFilter.Serializer();
 
-    @Parameterized.Parameter
```

Comment:
This leaves a few unused imports that will break CI

Reply:
done